decidim-dev/lib/decidim/dev/test/spec_helper.rb

```diff
@@ -27,6 +27,7 @@
   config.infer_spec_type_from_file_location!
   config.mock_with :rspec
   config.raise_errors_for_deprecations!
+  config.disable_monkey_patching!
```

Comment:
Nothing really, just a bit of consistency. But I did it wrong since I flipped the style instead of normalizing the current one :sweat_smile:. Let me update with a way less verbose commit :smiley:.